packages/ui/src/memberships/components/MemberListFilters/MemberListFilters.tsx

```diff
@@ -117,46 +118,31 @@ export const MemberListFilters = ({ roles, onApply }: MemberListFiltersProps) =>
         >
           <FounderMemberIcon />
         </TogglableIcon>
-
-        <ButtonPrimary onClick={onApply && (() => onApply(filters))}>Apply</ButtonPrimary>
+        <ButtonPrimary size="medium" onClick={onApply && (() => onApply(filters))}>
+          Apply
+        </ButtonPrimary>
       </Fields>
     </MemberListFiltersContainer>
   )
 }
 
-const ClearBtn = styled.button`
-  border: none;
-  margin: 0;
-  padding: 0.5rem;
-  width: auto;
-  overflow: visible;
-  cursor: pointer;
-  background: transparent;
-  color: ${Colors.Grey};
-  font: inherit;
-  line-height: normal;
-  appearance: none;
-  -webkit-font-smoothing: inherit;
-  -moz-osx-font-smoothing: inherit;
+const ClearButton = styled(ButtonBareGhost)`
+  position: absolute;
+  top: -32px;
+  right: 0;
 `
 
 const MemberListFiltersContainer = styled.div`
-  text-align: end;
+  display: flex;
+  position: relative;
 `
 
 const Fields = styled.div`
-  background: ${Colors.Black[100]};
-  display: flex;
+  display: grid;
+  grid-auto-flow: column;
```

Comment:
![Screenshot from 2021-05-17 20-35-41](https://user-images.githubusercontent.com/6571453/118541922-97634a00-b752-11eb-936a-6b80fab7825e.png)

@jodator with this display grid auto flow column I'm not sure how to align the apply button to the end (without using a position absolute). Which makes the layout harder to reuse for the past proposals filter box. Does it have to be a display grid ?

Reply:
OH, I see that now :disappointed:

It doesn't have to be `grid` IMO. The problem previously was with smaller screen sizes as on my mac's screen I couldn't reach the Apply button.

However, our designer is already working on changing the filters section here to a similar one introduced in the Proposals filtering.

Reply:
OH, I see that now :disappointed:

It doesn't have to be `grid` IMO. The problem previously was with smaller screen sizes as on my mac's screen I couldn't reach the Apply button.

However, our designer is already working on changing the filters section here to a similar one introduced in the Proposals filtering.

Reply:
Right I was just looking at the wired frames so I had not seen the different design for the past proposal filter :facepalm: .